.github/workflows/e2e-detox-template.yml

```
@@ -48,16 +59,20 @@ on:
 env:
   AWS_REGION: "us-east-1"
   ADMIN_EMAIL: ${{ secrets.MM_MOBILE_E2E_ADMIN_EMAIL }}
-  ADMIN_USERNAME: ${{ inputs.MM_TEST_USER_NAME || secrets.MM_MOBILE_E2E_ADMIN_USERNAME }}
-  ADMIN_PASSWORD: ${{ inputs.MM_TEST_PASSWORD || secrets.MM_MOBILE_E2E_ADMIN_PASSWORD }}
+  ADMIN_USERNAME: ${{ secrets.MM_MOBILE_E2E_ADMIN_USERNAME }}
+  ADMIN_PASSWORD: ${{ secrets.MM_MOBILE_E2E_ADMIN_PASSWORD }}
+  BRANCH: ${{ github.head_ref || github.ref_name }} 
+  COMMIT_HASH: ${{ github.sha }}
+  DEVICE_NAME: ${{ inputs.ios_device_name }}
+  DEVICE_OS_VERSION: ${{ inputs.ios_device_os_name }}
   DETOX_AWS_S3_BUCKET: "mattermost-detox-report"
   HEADLESS: "true"
   TYPE: ${{ inputs.run-type }}
   PULL_REQUEST: "https://github.com/mattermost/mattermost-mobile/pull/${{ github.event.number }}"
-  SITE_1_URL: ${{ inputs.MM_TEST_SERVER_URL || secrets.MM_MOBILE_E2E_SITE_1_URL }}
-  SITE_2_URL: ${{ secrets.MM_MOBILE_E2E_SITE_2_URL }}
-  SITE_3_URL: ${{ secrets.MM_MOBILE_E2E_SITE_3_URL }}
-  ZEPHYR_ENABLE: false
+  SITE_1_URL: "https://mobile-e2e-site-1.test.mattermost.cloud"
```

You removed the CMT functionality . We need to change the SITE_1 URL for the CMT tests 🚀 
```suggestion
  SITE_1_URL: ${{ inputs.MM_TEST_SERVER_URL || "https://mobile-e2e-site-1.test.mattermost.cloud" }} 
```

---

I've added this back